


icp_odometry:
  icp_objective: PointToPlane
  max_correspondence_dist: 0.25
  knn_normal_estimation: 4
  voxel_filter_size: -1.0
  max_n_iter: 25
  downsampling_ratio: 0.4
  crop_box_min_x: -25.0
  crop_box_min_y: -25.0
  crop_box_min_z: -100.0
  crop_box_max_x: 25.0
  crop_box_max_y: 25.0
  crop_box_max_z: 100.0
  
icp_mapping:
  map_voxel_size: 0.0
  min_movement_between_mapping_steps: 0.2
  min_refinement_fitness: 0.7
  scan_to_map_refinement:
    icp_objective: PointToPlane
    max_correspondence_dist: 3.0
    knn_normal_estimation: 5
    max_n_iter: 50
    voxel_filter_size: 0.0
    downsampling_ratio: 0.1
    crop_box_min_x: -30.0
    crop_box_min_y: -30.0
    crop_box_min_z: -100.0
    crop_box_max_x: 30.0
    crop_box_max_y: 30.0
    crop_box_max_z: 100.0
  map_builder:
    crop_box_min_x: -50.0
    crop_box_min_y: -50.0
    crop_box_min_z: -100.0
    crop_box_max_x: 50.0
    crop_box_max_y: 50.0
    crop_box_max_z: 100.0
    
local_map:
  voxel_size: 0.02
  crop_box_min_x: -12.0
  crop_box_min_y: -12.0
  crop_box_min_z: -100.0
  crop_box_max_x: 12.0
  crop_box_max_y: 12.0
  crop_box_max_z: 100.0
  

<<<<<<< HEAD
    
=======
mesher:
  is_compute_mesh: false
  voxel_size: 0.2
  knn_normal_estimation: 4
  strategy: Poisson # options: AlphaShape, Poisson, BallPivot
  alpha_shape_alpha: 0.5
  poisson_depth: 8
  poisson_min_density: -0.5
  poisson_scale: 0.8
  ball_pivot_radii: [0.5] 
    
>>>>>>> master
